src/app/products/manage-delinquency-buckets/delinquency-bucket/edit-bucket/edit-bucket.component.html

```diff
@@ -0,0 +1 @@
+<p>edit-bucket works!</p>
```

Comment:
@josehernandezfintecheandomx Edit bucket template is empty!

Reply:
Done!